app/views/visualizations/index.html.erb

```diff
@@ -67,9 +67,12 @@
               <% thumb = image_path("vis_#{type.downcase}_light.png")%>
             <% end %>
           <% end %>
-          <div class='item-image'
-               style='background-image:url(<%= thumb %>);background-size: 100% 100%'>
-          </div>
+          <a href='<%= visualization_path vi %>'>
```

Comment:
I know I asked for this, but could you instead try this:
```
<a href='<%=visualization_path vi %>'
     class='item-image'
     style='background-image:url(<%= thumb %>);
     background-size: 100% 100%'>
</a>
```

It looks to me as though the div is useless at the moment, unless it's being used for style purposes.